bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_Singlet_LO/ttH_Singlet_LO_madspin_card.dat

```diff
@@ -0,0 +1,30 @@
+#************************************************************
+#*                        MadSpin                           *
+#*                                                          *
+#*    P. Artoisenet, R. Frederix, R. Rietkerk, O. Mattelaer * 
+#*                                                          *
+#*    Part of the MadGraph5_aMC@NLO Framework:              *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#
+# set seed 1
+set Nevents_for_max_weigth 500 # number of events for the estimate of the max. weight
+# set BW_cut 15                # cut on how far the particle can be off-shell
+ set max_weight_ps_point 400  # number of PS to estimate the maximum for each event
+# 
+# specify the decay for the final state particles
+
+set max_running_process 1
+
+decay h > ta+ ta-
```

Comment:
Given that the madspin datacard includes the decay, probably is worth adding it to the filename...

Reply:
I mean the *higgs* decay

Reply:
You mean changing ttH_Singlet_LO to something like ttH_HtoTau_Singlet_LO ?
Is this ok?